assets/src/components/detours/activeDetourPanel.tsx

```diff
@@ -36,17 +40,19 @@ export const ActiveDetourPanel = ({
 
     <Panel.Body className="d-flex flex-column">
       <Panel.Body.ScrollArea>
-        {onNavigateBack && (
-          <Button
-            className="icon-link my-3"
-            variant="outline-primary"
-            onClick={onNavigateBack}
-          >
-            <ArrowLeft />
-            Back
-          </Button>
-        )}
-
+        <div className="d-flex flex-rown justify-content-between align-items-center">
```

Comment:
```suggestion
        <div className="d-flex flex-row justify-content-between align-items-center">
```

Reply:
😂 well I guess I don't need flex-row at all! It was not necessary
```suggestion
        <div className="d-flex justify-content-between align-items-center">
```